lib/galaxy/webapps/galaxy/api/datatypes.py

```diff
@@ -112,6 +116,15 @@ async def converters(self) -> DatatypeConverterList:
         """Gets the list of all installed converters."""
         return view_converters(self.datatypes_registry)
 
+    @router.get(
+        '/api/datatypes/suitable_converters',
+        summary="Returns the list of converters that are suitable for an array of datatypes",
+        response_description="List of suitable converters"
+    )
+    async def suitable_converters(self, trans, id, instance_type='history') -> List[dict]:
```

Comment:
Everything that touches the database (so everything basically 😢 ) must be a sync route currently.
```suggestion
   def suitable_converters(self, trans, id, instance_type='history') -> List[dict]:
```